packages/server/server.ts

```diff
@@ -76,10 +86,33 @@ uws
     idleTimeout: 0,
     maxPayloadLength: 5 * 2 ** 20,
     upgrade: handleUpgrade,
-    open: handleOpen,
-    message: handleMessage,
-    // today, we don't send folks enough data to worry about backpressure
-    close: handleClose
+    open: (ws: WebSocket<any>) => {
+      incrementWebSocketConnections()
+      trackWebSocketOpen(ws)
```

Comment:
-1 Why have a setting `ENABLE_METRICS` if we're collecting metrics unconditionally? Instead we should conditionally bind the function here. I suggest something along the lines of
```ts
// at the top of the file
const metricsEnabled = process.env.ENABLE_METRICS === 'true'
```
```ts
// here
open: !metricsEnabled ? handleOpen : (ws) => {
  trackOpen(ws)
  handleOpen(ws)
}
```
```ts
// metrics.ts
export const trackOpen = ((ws: WebSocket) => {
    incrementWebSocketConnections()
    trackWebSocketOpen(ws)
}
```
This way we don't need to know how to track things here and we don't spend any effort if tracking is disabled.